### ISSUE TITLE
Missing filtrationMode bloats Loki labels

### ISSUE THREAD

**Issue by AntonPetrov83** [serilog-contrib/serilog-sinks-grafana-loki#81]
### Which version of Serilog.Sinks.Grafana.Loki are you using?

7.1.0

### Which version of .NET are you using?

net5.0

### Describe the bug

I left label filtration mode uninitialized during configuration and received labels I was not expected. This is because of current implementation:
```
private bool IsAllowedByFilter(string label) =>
            _filtrationMode switch
            {
                LokiLabelFiltrationMode.Include => IsInFilterList(label),
                LokiLabelFiltrationMode.Exclude => !IsInFilterList(label),
                null => true,
                _ => true
            };
```

I think this is a very bad design decision because overusing labels downgrades Loki performance. I suggest that the default behaviour of `IsAllowedByFilter` should be to return `false`. 

Currently it looks like it is not possible to ignore label filtration logics at all.

### To Reproduce

-

### Expected behavior

-

### I have read the documentation

- [X] I have read [Common Problems](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/wiki/Common-problems) section in project wiki

**Comment by mishamyte** [comment]
The current behavior is works as expected, as for me. 

1. The labels are important for Loki, this is described in wiki
2. If there are no filtration mode it is oblivious that all data would be returned (it is the common behavior for filters anywhere. If you won't specify `Where` in EF - it won't do a filtration)
3. Changing behavior to your version will increase a count of issues kind of "Sink is not working!1"

So I don't think this is the way we're going. 

Facing the performance issues, related to not using labels is a good point to go the Loki + this sink doc and read it

**Comment by AntonPetrov83** [comment]
1. Agree. And you say it in your docs: "Big count of streams can kill Loki and called high cardinality." - in my opinion a default configuration of a sink must not "kill" anything ))) that is why I created this issue. We should avoid creating dangerous defaults.

2. Can you imagine any case when `filtrationMode = null` will be a desired behaviour? I doubt it and right now this is a default value. Because filtration implies some performance impact I would make this entire feature to be optional. Would be great to say "I do not need automatic labeling at all."

3. Understood. I guess this is because some labels are required by Loki. Then `LokiSink` should throw an exception if no labels were configured.

Thank you for your time answering this 👍

**Comment by mishamyte** [comment]
@AntonPetrov83 the change of behavior was implemented in [v8.0.0-beta.0](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/releases/tag/v8.0.0-beta.0)

Now you explicitly specify what properties will be mapped to the labels. Rest will be appended in the body (by default formatter). If you will define no list - no labels would be created.

More details in #99
